### DISCUSSION TITLE
can i copy past plot files from one drive to another?

### DISCUSSION THREAD

**Discussion by Money-for-nothing** [Chia-Network/chia-blockchain#3089] [Q&A]
one drive is Full and i like to transfer the plot files to another drive is it possible to copy paste them ?

**Comment by Cast39** [comment]
yes, this is no problem. For me it also worked while farming but just to be sure you can stop chia and restart when you are finished

**Comment by Money-for-nothing** [comment]
hmm i have a problem still.. i copied the plots to the new drive but the Chia doesn't recognise them and thinks they ar corrupt.. the old ones are still working though i don't want to delete the originals before i have confirmed info they are functioning..

**Reply by Cast39** [reply]
you can remove the original directory from your chia instance and just add the new one. If this works you can delete the old ones. If this does not work you might need to copy them again. Is your new Drive connected to the same Chia instance (aka with the same key)?

**Reply by Team4N6** [reply]
Like Cast said, you can only copy plots if you're using the same key. Your key is hard coded into the plot. So you can't move plots between computers if they are using two different keys. However, you should be able to copy them to a new drive on the same computer. I had a problem with a plot that I had to recopy. Apparently writing TBs of data at a time isn't always perfect.
